core/lib/i18n/i18n.js

```diff
@@ -202,6 +202,8 @@ function createIcuMessageFn(filename, fileStrings) {
     const unixStyleFilename = filenameToLookup.replace(/\\/g, '/');
     const i18nId = `${unixStyleFilename} | ${keyname}`;
 
+    message = escapeIcuMessage(message);
```

Comment:
Should we call this inside `formatMessage`. There is another usage of `formatMessage` in the flow report renderer btw.